components/Loading/index.js

```diff
@@ -8,14 +8,32 @@ const Overlay = styled.div`
   align-items: center;
 `;
 
+const SpinAnimation = css`
+@keyframes spin-animation {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+`
+
 const Content = styled.div`
-  font-size: 2rem;
+  width: 3rem;
+  height: 3rem;
+  border: 1rem solid;
+  border-color: var(--mid-blue) transparent var(--light-blue) transparent;
+  border-radius: 50%;
+  animation: spin-animation 1.2s linear infinite;
+  ${SpinAnimation}
 `;
 
+
 export default function Loading() {
   return (
     <Overlay>
-      <Content>Loading...</Content>
+      <Content></Content>
```

Comment:
I think this can now be a self closing tag. 

```suggestion
      <Content />
```